src/services/user-service.js

```diff
@@ -1,39 +1,51 @@
+const LRUCache = require('lru-native');
 const { log } = require('steno');
 
 const User = require('../models/UserModel');
-const cache = require('./cache-service.js');
 
-module.exports = {
-    get(uid) {
-        const cachedUser = cache.get(uid);
+const cache = new LRUCache({
+    maxElements: 1000,
+    maxAge: 120 * 60 * 1000, // 2 hours
+    size: 100
```

Comment:
imo, live cache recalculation is overcomplicated for now, so the static default config is not bad idea for now, right?